### ISSUE TITLE
False positive: cannot assign scalar variable to composite target

### ISSUE THREAD

**Issue by rts-barry** [okbob/plpgsql_check#16]
Assigning untyped 'null' to a composite target makes plpgsql_check detect an error where there is none.

```sql
CREATE TABLE t(i int);

CREATE FUNCTION test_t(OUT t) RETURNS t AS $$
BEGIN
	$1 := null; -- null::t would prevent error reporting
END;
$$ LANGUAGE plpgsql;

LOAD 'plpgsql';
SELECT plpgsql_check_function_tb('public.test_t()');

-- "(test_t,3,assignment,42804,"cannot assign scalar variable to composite target",,,error,,,)"
```

**Comment by okbob** [comment]
Should be ok now, thank you for report

**Comment by rts-barry** [comment]
It works, thank you very much for your fix.
